### DISCUSSION TITLE
Installation bricked my Acer Swift laptop

### DISCUSSION THREAD

**Discussion by Universal-Solder** [elementary/triage#572] [Q&A]
I downloaded Elementary, etched it on a USB stick, started my laptop, selected the USB stick (F12) to boot from, and installed Elementary with the option to delete everything and install the new system.
After it claimed to have finished installation, it asked me to remove the installation medium (USB stick) and press Enter.
To my surprise, my laptop woke up to the Debian Linux that was on it before, and asked me for a password, that I couldn't provide.
Since nothing else worked, I tried hard-resetting my laptop by pressing the power button. It worked before, that long pressing (maybe like 20 seconds) rebooted my laptop. This is no longer working. Regardless of how long I press the power button or any other button, the power LED is flashing and nothing else ever happens. 
Now I have a beautiful slim-line aluminum brick. Sweet.

**Comment by jeremypw** [comment]
According to https://community.acer.com/en/kb/articles/75-acer-internal-battery-reset  you need to disconnect the AC adapter as well. I thought the hard reset was a hardware thing that user software couldn't override but I may be wrong.  On some Acers there is a pin-hole as well apparently that causes a reset.  

Which version of elementary did you install? Did you choose disk encryption?

**Reply by Universal-Solder** [reply]
The pinhole! I didn't realize it was there. It unbricked the laptop, and I'm trying to install again! Thank you so much.
